private/rules/coursier.bzl

```diff
@@ -963,6 +965,12 @@ def rewrite_files_attribute_if_necessary(repository_ctx, dep_tree):
 
     return dep_tree
 
+def is_dep(jar_path, deps):
```

Comment:
I think you can inline this function.

Reply:
Hmm, doesn't seem to be easier to read. After some tweak, this PR would be more or less like #1269 anyway, so maybe we can take it from there?